tests/test_client.py

```diff
@@ -380,13 +380,18 @@ def test_get_jobs(self, fakerequest, fakeclones, simple_config):
 
     def test_client_errors(self):
         """Test creation of exceptions"""
-        err = oqe.RequestError("GET", "http://localhost", 404)
+        template_error = (
```

Comment:
I suggest to use a short dummy text here instead. Then no line splitting is needed

Reply:
yeah, agreed. I do always like "real" data in tests when possible, but if it's just an arbitrary string, no need to cause ourselves pain.